src/scripts/util.sh

```diff
@@ -234,7 +234,7 @@ parse_config_file() {
         # for the certbot command. This domain will be your Common Name on the
         # certificate.
         # stackoverflow on this awk usage: https://stackoverflow.com/a/45808487
-        certs["${cert_name}"]="$(echo ${certs["${cert_name}"]} "${server_names[@]}" | xargs -n1 echo | awk '!a[$0]++' | tr '\n' ' ')"
+        certs["${cert_name}"]="$(echo "${certs["${cert_name}"]}" "${server_names[@]}" | xargs -n1 echo | awk '!a[$0]++' | tr '\n' ' ')"
```

Comment:
I am a bit skeptical about adding quotes around this expansion, but I think the tests cover it. Will double check tomorrow.